### PR TITLE
Added Python wrapper for fin and also regression test

### PR DESCRIPTION
**Pull request by macd**
Signed-off-by: Don MacMillen <don@macmillen.net>
There were no regression tests for this so I added one tcl and one Python test.

### CODE REVIEW

File: src/fin/test/gcd_fill.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from openroad import Design, Tech
+import odb
+
+tech = Tech()
+design = Design(tech)
+
+design.evalTclString("read_db gcd_fill.odb; read_sdc gcd_fill.sdc")
```
Comment (maliberty) [inline src/fin/test/gcd_fill.py:7]: Why read_db in tcl eval and not through the design object?

Why do we need timing for metal fill?

File: src/fin/test/gcd_fill.tcl [hunk 2]
```diff
@@ -0,0 +1,10 @@
+read_db gcd_fill.odb
+read_sdc gcd_fill.sdc
+
+set_propagated_clock [all_clocks]
```
Comment (maliberty) [inline src/fin/test/gcd_fill.tcl:4]: same - timing not needed

**Review by maliberty — COMMENTED** [review]
Thanks for adding a test.  It was really needed.

File: src/fin/test/gcd_fill.tcl (hunk 2 above)
Comment (macd) [reply src/fin/test/gcd_fill.tcl:4]: You are right, we don't need the timing. I was just doing copy paste from flow scripts to make a test case. Will remove.

Comment (macd) [reply src/fin/test/gcd_fill.tcl:4]: And therefore we  don't need the sdc either.

File: src/fin/test/gcd_fill.py (hunk 1 above)
Comment (macd) [reply src/fin/test/gcd_fill.py:7]: So readDb is not yet on the design object, which I guess, it should be. Will do

**Review by maliberty — APPROVED** [review]

**Review by maliberty — CHANGES_REQUESTED** [review]
You need to add fin to test/regression so this test is run with the others.

**Comment by macd** [comment]
Ah, yes, of course. Will do.

**Review by maliberty — APPROVED** [review]

**Pull request merged**